iib/workers/tasks/opm_operations.py

```diff
@@ -193,6 +193,134 @@ def _serve_cmd_at_port(serve_cmd, cwd, port, max_tries, wait_time):
     raise IIBError(f'Index registry has not been initialized after {max_tries} tries')
 
 
+def _get_temp_index_db_fbc(base_dir, from_index):
```

Comment:
should from_index be optional? Since you are checking for the presence of that param?

Reply:
+1, and there is also creation index.db file if from_index is not set

Reply:
I changed this function a lot, please check if it is better now.